rad/prelude/error-messages.rad

```diff
@@ -273,3 +275,15 @@ This may be because you have no internet connection, you are not authorized to p
 (:test "missing-key-file"
   [(missing-key-file) ==> "Missing file: my-keys.rad
 Do `rad key create` to create your own key pair and rerun your command."])
+
+(def rad-ipfs-key-gen-failure
+  (fn [stderr]
+    (string-append
+      "error: Failed to create key for Git IPFS repository:\n"
+      stderr)))
+
+(def rad-ipfs-name-publish-failure
+  (fn [stderr]
+    (string-append
+      "error: Failed to publish key for Git IPFS repository:\n"
+      stderr)))
```

Comment:
Please add tests and a documentation on where the errors occur and when.

Reply:
I added documentation but I’m not a big fan of adding tests here. The logic is so trivial that the tests are basically copy-and-pasting the function body. This does not prevent errors and just adds maintenance overhead.

Reply:
If someone not so familiar with radicle is supposed to update the texts, I think it would be good for them to have a quick way of verifying that the functions are still working and give the proper results.

Reply:
Can you think of a better way for this scenario?

Reply:
Ah, you’re right. I didn’t consider people not familiar with Radicle. I added the tests